content/ieco/0004-bancos-moeda-crises-financeiras.md

```diff
@@ -129,24 +177,95 @@ Para resolver este conflito de interesses, existem múltiplas opções:
+    dizendo que o preço para um mesmo bem se deve tentar igualar entre países
+    (excluindo custos de transporte).
+    - Uma das consequências a curto prazo da globalização é a **convergência dos preços**
+      para a média mundial.
+  - A **Balança de Pagamentos** regista as fontes e a utilização e moeda estrangeira:
+    - **Investimento de portfólios**: compra de ações/títulos estrangeiros;
+    - **Investimento direto estrangeiro**: propriedade de ativos físicos estrangeiros.
+- **[Mercados de trabalho internacionais](color:orange)**: não tem sido tão afetado, tendo em conta que os
+  salários ainda diferem muito entre países.
+
+### Vantagens e Desvantagens
+
+A globalização traz benefícios ou malefícios consoante a **escassez relativa de fatores**:
+
+- Um fator **abundante** dentro de um país é barato, mas se abrir ao resto do mundo,
+  o seu preço **aumenta**, convergindo para a média mundial;
+- Por outro lado, um fator **escasso** é caro, mas vai convergir para um preço mais **barato**.
+
+Assim, a existência de um mercado livre global tende para a **especialização** na produção.
+
+::::info[Especialização]
+
+Imaginemos um país abundante em capital e outro país abundante em mão-de-obra.
+
+Com a abertura dos mercados, cada país especializa-se nos seus fatores abundantes.
+
+No país com abundância de capital, os donos desse capital ganham mais com o mercado global do
+que os trabalhadores, aumentando a desigualdade.
+
+No país com abundância de mão-de-obra, são os trabalhadores que ganham mais com a globalização,
+diminuindo a desigualdade.
+
+::::
+
+Um país tem **vantagem comparativa** na produção de um bem se a sua produção possuir
+**menor custo de oportunidade**.
+
+A especialização de acordo com a vantagem comparativa tem efeitos semelhantes aos do progresso tecnológico,
+tais como o aumento da produtividade, a diminuição de emprego a curto prazo e o crescimento das indústrias
+de exportação a longo prazo.
+
+::::info[Vantagem comparativa e Vantagem absoluta]
+
+Imaginemos dois estudantes, o Afonso e o Miguel, que realizam um trabalho de grupo:
+
+- O Afonso demora 3 horas a fazer a parte 1 do trabalho e 4 horas a fazer a parte 2;
+- O Miguel demora 5 horas a fazer a parte 1 do trabalho e 6 horas a fazer a parte 2;
+
+O Afonso tem vantagem absoluta nas duas partes do trabalho (menos horas de trabalho), mas tem vantagem
+comparativa na parte 1 porque:
+
+- 3/4 = 0.75 < 5/6 = 0.833, ou seja, para fazer a parte 1 o Afonso sacrifica 75% do tempo de realização
+  da parte 2 (custo de oportunidade de 1 em termos de 2), enquanto que o Miguel sacrifica 83.3%.
+- 4/3 = 1.33 > 6/5 = 1.2, ou seja, o Afonso tem o custo oportunidade maior de fazer a parte 2 do
+  trabalho comparativamente ao Miguel.
+
+Logo o Afonso deve fazer a parte 1 e o Miguel a parte 2.
+
+::::
+
+### Intervenção dos Governos
+
+- **Imposição de tarifas**: discriminização de produtos estrangeiros.
```

Comment:
`discriminização` is not a word, perhaps `discriminação` or `descriminalização`?